lib/sycamore/sycamore/utils/pdf_utils.py

```diff
@@ -162,3 +162,23 @@ def display_page_and_table_properties(some_pages: list[Document]):
                 print("Element Type: ", e.type)
                 print("Element Properties: ", json.dumps(e.properties, indent=2, default=str))
                 display(HTML(e.text_representation))
+
+
+def promote_sectionheader_to_title(elements: list[Element]) -> list[Element]:
```

Comment:
Why don't we modify this to promote any set of elements to a title and take it in as an argument? You can set the default to be `["Section-header", "Caption"]` and call the function `promote_elements`. This would make it more widely applicable.

Reply:
good call, updating the function